doc/install.rst

```diff
@@ -38,178 +19,27 @@ Scikit-learn requires:
-release::
+    conda install scikit-learn
 
-    pip install --user --install-option="--prefix=" -U scikit-learn
-
-The ``--user`` flag asks pip to install scikit-learn in the ``$HOME/.local``
-folder therefore not requiring root permission. This flag should make pip
-ignore any old version of scikit-learn previously installed on the system while
-benefiting from system packages for numpy and scipy. Those dependencies can
-be long and complex to build correctly from source.
-
-The ``--install-option="--prefix="`` flag is only required if Python has a
-``distutils.cfg`` configuration with a predefined ``prefix=`` entry.
-
-
-From source package
-~~~~~~~~~~~~~~~~~~~
-
-Download the source package from http://pypi.python.org/pypi/scikit-learn/
-, unpack the sources and cd into the source directory.
-
-This packages uses distutils, which is the default way of installing
-python modules. The install command is::
-
-    python setup.py install
-
-or alternatively (also from within the scikit-learn source folder)::
-
-    pip install .
-
-.. warning::
-
-   Packages installed with the ``python setup.py install`` command cannot
-   be uninstalled nor upgraded by ``pip`` later. To properly uninstall
-   scikit-learn in that case it is necessary to delete the ``sklearn`` folder
-   from your Python ``site-packages`` directory.
-
-
-.. _install_by_distribution:
-
-Third party distributions of scikit-learn
-=========================================
-
-Some third-party distributions are now providing versions of
-scikit-learn integrated with their package-management systems.
-
-These can make installation and upgrading much easier for users since
-the integration includes the ability to automatically install
-dependencies (numpy, scipy) that scikit-learn requires.
-
-The following is an incomplete list of Python and OS distributions
-that provide their own version of scikit-learn.
-
-
-Debian and derivatives (Ubuntu)
--------------------------------
-
-The Debian package is named ``python-sklearn``
-(formerly ``python-scikits-learn``)
-and can be installed using the following command::
-
-      sudo apt-get install python-sklearn
-
-Additionally, backport builds of the most recent release of
-scikit-learn for existing releases of Debian and Ubuntu are available
-from the `NeuroDebian repository
-<http://neuro.debian.net/pkgs/python-sklearn.html>`_ .
-
-A quick-'n'-dirty way of rolling your own ``.deb`` package
-is to `use stdeb <https://github.com/scikit-learn/scikit-learn/wiki/Quick-packaging-for-Debian-Ubuntu>`_.
+Third-party Distributions
+==========================
+If you don't already have a python installation with numpy and scipy, we
+recommend to install either via your package manager or via a python bundle.
+These come with numpy, scipy, scikit-learn, matplotlib and many other helpful
+scientific and data processing libraries.
 
+**We don't recommend installing scipy or numpy using pip on linux**,
```

Comment:
Yeah it should be up there. What solution? Installing a third-party distribution?